hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/TestCloseContainerByPipeline.java

```diff
@@ -73,8 +75,11 @@
   public static void init() throws Exception {
     conf = new OzoneConfiguration();
     conf.set(ScmConfigKeys.OZONE_SCM_PIPELINE_OWNER_CONTAINER_COUNT, "1");
+    conf.setInt(OZONE_DATANODE_MAX_PIPELINE_ENGAGEMENT, 2);
```

Comment:
Should we expose a method for MiniOzoneCluster to set per DN pipeline limit?

Reply:
Conf setting is universal way of setting the limit. The current way in MiniOzoneCluster to set the global pipeline limit is rather a convenient method for internal use.

Reply:
I'm fine with current approach. I'm asking because I see a few other mini ozone cluster based tests require similar settings, adding a wrapper can make this easier. But we can do that as a follow up jira.